cit3.0-web/src/components/UserTable/UserTable.js

```diff
@@ -0,0 +1,207 @@
+import { useEffect, useState } from "react";
+import Container from "react-bootstrap/Container";
+import { v4 } from "uuid";
+import Row from "react-bootstrap/Row";
+import Col from "react-bootstrap/Col";
+import "./UserTable.css";
+import { Button } from "shared-components";
+import { Modal, Button as BTN } from "react-bootstrap";
+import UserListItem from "../UserListItem/UserListItem";
+import UserFactory from "../../store/factory/UserFactory";
+import { useKeycloakWrapper } from "../../hooks/useKeycloakWrapper";
+import {
+  deleteUser,
+  getUsers,
+  updateUserAssignments,
+} from "../../store/actions/user";
+
+const UserTable = () => {
+  const keycloak = useKeycloakWrapper();
+
+  const [isDelete, setDelete] = useState(false);
+  const [show, setShow] = useState(false);
+  const [label, setLabel] = useState("");
+  const [body, setBody] = useState("");
+  const [users, setUsers] = useState([]);
+  const [userToUpdate, setUserToUpdate] = useState({});
+  const handleShow = () => setShow(true);
+  const handleClose = () => {
+    setShow(false);
+  };
+
+  const handleRemoveMunicipality = (index) => {
+    userToUpdate.municipalities = userToUpdate.municipalities.filter(
+      (_, ind) => ind !== index
+    );
+    setUserToUpdate({ ...userToUpdate });
+  };
+
+  const handleRemoveRegional = (index) => {
+    userToUpdate.regionalDistricts = userToUpdate.regionalDistricts.filter(
+      (_, ind) => ind !== index
+    );
+    setUserToUpdate({ ...userToUpdate });
+  };
+
+  const handelEditAction = (user) => {
```

Comment:
Small typo here, "handel" instead of "handle"

Reply:
Yup :)